src/tasks/sep/042-U17-sep-base/config.toml

```diff
@@ -0,0 +1,48 @@
+l2chains = [
+    {name = "Base Sepolia Testnet", chainId = 84532}
+]
+
+templateName = "OPCMUpgradeV500"
+
+allowOverwrite = ["SuperchainConfig"]
+
+
+
+# Base Sepolia Testnet (Permissionless)
+
+[[opcmUpgrades]]
+chainId = 84532
+absolutePrestate = "0x03799051d2bfe459127d4597f469f535ff1bd2a6e1e2134443167620871c11f3"  # prestates."1.8.0-rc.3" https://github.com/ethereum-optimism/superchain-registry/blob/c9881d543174ff00b8f3a9ad3f31bf4630b9743b/validation/standard/standard-prestates.toml#L6
+expectedValidationErrors = "PROXYA-10,DF-30,PDDG-DWETH-30,PDDG-VM-30,PDDG-130,PLDG-DWETH-30,PLDG-VM-30"
```

Comment:
Why does base have more expected validation errors than the other tasks? It should be the same if we're using the validation overrides properly